go/account/templates/account/account_summary_mail.txt

```diff
@@ -20,8 +20,8 @@ Totals per conversation type:
 {{type}}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 {% for conversation in conversations %}{{conversation.name}}
-    Sent: {{conversation.count_sent_messages}} to {{conversation.count_outbound_uniques }} uniques.
-    Received: {{conversation.count_replies}} from {{conversation.count_inbound_uniques }} uniques.
+    Sent: {{conversation.count_outbound_messages}} to {{conversation.count_outbound_uniques }} uniques.
+    Received: {{conversation.count_inbound_messages}} from {{conversation.count_inbound_uniques }} uniques.
```

Comment:
I can't find ``count_outbound_messages`` or ``count_inbound_messages`` in the vumi-go code?